modules/gdscript/gdscript.cpp

```diff
@@ -164,7 +165,7 @@ GDScriptInstance *GDScript::_create_instance(const Variant **p_args, int p_argco
 			MutexLock lock(GDScriptLanguage::singleton->mutex);
 			instances.erase(p_owner);
 		}
-		ERR_FAIL_V_MSG(nullptr, "Error constructing a GDScriptInstance.");
+		ERR_FAIL_V_MSG(nullptr, "Error constructing a GDScriptInstance: " + Variant::get_call_error_text(instance->owner, "@implicit_new", nullptr, 0, r_error));
```

Comment:
is this something I should do?

Reply:
Also, does err_v stop the script? If it does, it's good to keep the memory freeing here before the message, to not cause a memory leak.

Reply:
Yes, `ERR_FAIL_*` macros have `return`. I think you should use a variable:

```cpp
String error_text = Variant::get_call_error_text(...);
instance->owner->set_script_instance(nullptr);
...
ERR_FAIL_V_MSG(nullptr, "Error constructing a GDScriptInstance: " + error_text);
```

Reply:
I see, so you're suggesting assigning the variable to grab the message before the script instance is removed, so that we make sure we get the right error?

Reply:
I think so, in this case the error text will contain the path to the script, judging by the snippets that I have given above.

Reply:
Seems like a good idea. I changed that just now.